src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.cpp

```diff
@@ -588,6 +588,10 @@ size_t jit_store_emitter::aux_vecs_count() const {
         (src_prc_ == Precision::FP32 && dst_prc_ == Precision::BF16))
         count++;
 
+    // for data swapping to avoid using Xmm(0) as I/O xmm for jit_emu_vcvtneps2bf16
```

Comment:
Special logic for SSE have to be hided inside `jit_emu_vcvtneps2bf16` itself. Otherwise we will need to copypaste everywhere the emitter is used.

Reply:
To be honest I'm not really sure that it would be better.
Firstly, as I understand `SSE` emitters doesn't support xmm0 as input/output. So user has to manually choose correct registers. But in `store_emitter` user cannot choose explicitly I/O registers for `jit_uni_vcvtneps2bf16`. Because it's hidden by `store_emitter` logic. 
Secondly, `store_emitter` is derived class of "jit_emitter" that explicitly and always choose `xmm_0` as `aux_vmm_regs[0]` for `SSE`.
Thus, `store_emitter` should resolve this problem as user resolves this problem for other implementations

As for other cases when we use `jit_uni_vcvtneps2bf16` user has to manually choose I/O registers. And it's OK. We will get the same problems if we have the nesting emitters (as this case with `store_emitter`). For example, it would be valid for Snippets but they doesn't support `SSE` as we know.

Also if we want to move special logic for `SSE` into `jit_uni_vcvtneps2bf16` how we can cover `xmm_0` as  I/O case? When `preamble` is called we will get exception that `xmm_0` is I/O registers for `SSE`. Honestly We can override `emit_code` for `jit_uni_vcvtneps2bf16` for example but I'm not sure that this idea is better.

What do you think?

Reply:
I am ok to go with current impl in bounds of this task.

In general I understand there is a limitation on IO vecs to be not equal XMM0 for all emitters, and in fact that seems be not correct. How I see it: there shouldn't be any limitations on set of registers used as I/O for emitters. If some emitter uses is forced to use XMM0 is should be handled inside. In any case that could be handled by spills on stack and base jit_emitter class should provide unified utils for that.
Anyway that could be resolved as separate task.